engine/src/main/java/org/destinationsol/menu/MainMenuScreen.java

```diff
@@ -57,25 +58,25 @@
         displayDimensions = SolApplication.displayDimensions;
 
         tutorialControl = new SolUiControl(buttonWidth, buttonHeight, UiDrawer.positions.get("bottom"), 0, calculateButtonOffsetFromBottom(3), true, Input.Keys.T);
-        tutorialControl.setDisplayName("Tutorial");
+        tutorialControl.setDisplayName(Translation.translate("${core:titlemenu#tutorial}"));
```

Comment:
Instead of having to `Translation.translate("${GESTALT_ID}")` every time, can't we simply pass `GESTALT_ID` to `setDisplayName(GESTALT_ID)` which will then call the translation itself?
I think that it should be up to the function that actually does the text display to handle the translation thus the calls to it only need to provide where to look for said translation.
NOTE that this might mean changing the name of the `setDisplayName(...)` function to something more appropriate, like `setDisplayTranslationString()` or something

Reply:
This is the way tera does it. Also, said changes would need to happen in uiOverhaul branch.

Reply:
As Adrijaned had said, according to the wiki https://metaterasology.github.io/docs/developing/translation.html that is the method Terasology has in translation. That definitely sounds like that would be more straightforward though.